### PR TITLE
fix: release bump for the SDK

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

Release "0.32.0".

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [v0.31.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.31.0) (July 28, 2023)
```
Comment (wtrocki) [inline CHANGELOG.md:3]:
```suggestion
## [v0.32.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.31.0) (July 28, 2023)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [v0.32.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.31.0) (July 28, 2023)
+
+[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.31.0...v0.31.0)
```
Comment (tibulca) [inline CHANGELOG.md:5]:
```suggestion
[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.31.0...v0.32.0)
```
show it be like this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [v0.32.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.31.0) (July 28, 2023)
```
Comment (tibulca) [inline CHANGELOG.md:3]:
```suggestion
## [v0.32.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.32.0) (July 28, 2023)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+**Breaking changes:**
+
+- fix: Removed omitempty from tags in ServerlessUpdateRequestParams [\#509](https://github.com/mongodb/go-client-mongodb-atlas/pull/509) ([cveticm](https://github.com/cveticm))
+- INTMDB-545!: add Azure support to cloudProviderAccess [\#507](https://github.com/mongodb/go-client-mongodb-atlas/pull/507) ([andreaangiolillo](https://github.com/andreaangiolillo))
+
+**Merged pull requests:**
+
+- INTMDB-545: Update CloudProviderAccessService to support azure [\#508](https://github.com/mongodb/go-client-mongodb-atlas/pull/508) ([andreaangiolillo](https://github.com/andreaangiolillo))
+
+## [v0.31.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.31.0) (July 06, 2023)
```
Comment (tibulca) [inline CHANGELOG.md:16]: I see another v0.31.0 entry at line 25, should these be merged?

**Comment by wtrocki** [comment]
I followed the official release process but got some strange results. with duplicated version

File: .github_changelog_generator [hunk 5]
```diff
@@ -1,4 +1,4 @@
-future-release=v0.31.0
+future-release=v0.32.0
```
Comment (tibulca) [inline .github_changelog_generator:1]: can you please check if this should be 0.33.0?
Maybe this is why you had this issue: `I followed the official release process but got some strange results. with duplicated version`

**Comment by wtrocki** [comment]
> can you please check if this should be 0.33.0?

Changelog seems ok now. Added new step to release process.

File: .github_changelog_generator [hunk 6]
```diff
@@ -1,4 +1,4 @@
...
 since-tag=v0.30.0
```
Comment (andreaangiolillo) [inline .github_changelog_generator:2]: you need to update this as well

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**